### DISCUSSION TITLE
Forms Builder

### DISCUSSION THREAD

**Discussion by nithya-picco** [spurtcms/spurtcms#87] [General]
We are excited in introducing a powerful feature in spurtcms the concept of Forms Builder.
![form](https://github.com/user-attachments/assets/0ca54f3c-bdba-4f6f-a4ce-c0d054fbedcf)

**Comment by spurtcmsdeveloper-73** [comment]
The Forms Builder in SpurtCMS will undoubtedly enhance user experience by allowing for easy and customizable form creation.It’s great to see such powerful tools.
